python/cuvs/cuvs/neighbors/ivf_flat/ivf_flat.pyx

```diff
@@ -30,9 +30,10 @@ from cuvs.distance_type cimport cuvsDistanceType
 from pylibraft.common import auto_convert_output, cai_wrapper, device_ndarray
 from pylibraft.common.cai_wrapper import wrap_array
 from pylibraft.common.interruptible import cuda_interruptible
-from pylibraft.distance.pairwise_distance import DISTANCE_TYPES
 from pylibraft.neighbors.common import _check_input_array
```

Comment:
We'll probably need to bring this over at some point too (no rush for 24.06 since we already depend on `pylibraft`.